quickstart-sda.md

```diff
@@ -0,0 +1,239 @@
+
+NeIC Sensitive Data Archive documentation can be found at: https://neic-sda.readthedocs.io/en/latest/ .
+
+## Deployment
+
+The storage and interfaces stack can be deployed with the use of the provided `docker-compose.yml` file by running
+
+```cmd
+docker compose up -d
+```
+
+from the root of this repo. Configuration can be customized by changing the `config/config.yml` file.
+
+### Adding TLS to internet facing services
+
+Internet facing services such as e.g. `s3inbox`, `download` and `mock-oidc`, need to be secured via TLS certification. This can be most conveniently achieved by using [letsencrypt](https://letsencrypt.org/getting-started/) as Certificate Authority. Assuming shell access to your web host, a convenient way to set this up is through  installing certbot (or any other ACME client supported by letsencrypt). Detailed instructions on setting up `certbot` for different system configurations can be found [here](https://certbot.eff.org/).
+
+## Authentication for users (mock or AAI)
+
+To interact with SDA services, users need to provide jw token authentication. Ultimately, tokens can be fetched by [LS-AAI](https://lifescience-ri.eu/ls-login/) upon user login to an OIDC relaying party (RP) service that is [registered with LS-AAI](https://spreg-legacy.aai.elixir-czech.org/). An example of such an RP service is the [sda-auth](https://github.com/neicnordic/sda-auth) service, which however not included in the present stack.
+
+For the purposes of the starter-kit, jw tokens can be issued by the included `mock-oidc` service and used for authentication, instead. The `mock-oidc` is a simple python implementation that mimics basic OIDC functionality of LS-AAI. It doesnot require user authentication and serves a valid token through its `/token` endpoint:
+
+```cmd
+token=$(curl -s -k https://<mock-oidc_domain_name>/tokens | jq -r '.[0]')
+```
+
+This token is created upon deployment, see `scripts/make_credentials.sh` for more details. Note that the api actually returns a list of tokens where the first element is the token of interest and the rest are legacy tokens for [testing  `sda-download`](https://github.com/neicnordic/sda-download/blob/main/dev_utils/README.md#get-a-token).
+
+## How to perform common user tasks
+
+### Data encryption
+
+The `sda-pipeline` only ingests files that are encrypted with the archive's `c4gh` public key.For instance, using the go implementation of the [crypt4gh utility](https://github.com/neicnordic/crypt4gh) a file can be encrypted simply by running:
+
+```bash
+crypt4gh encrypt -f <file-to-encrypt> -p <sda-c4gh-public-key>
+```
+
+
+### Uploading data
+
+Users can upload data to the SDA by transferring them directly to the archive's `s3inbox` with an S3 client tool such as [s3cmd](https://s3tools.org/s3cmd):
+
+```bash
+s3cmd -c s3cmd.conf put <path-to-file.c4gh> s3://<username>/<target-path-to-file.c4gh>
+```
+
+where `s3cmd.conf` is a configuration file with the following content:
+```json
```

Comment:
This isn't JSON but either in "INI" or TOML format. I suggest using tho `ini` language tag.

```suggestion
```ini
```